packages/editor/src/bindings/post-meta.js

```diff
@@ -1,8 +1,8 @@
 /**
  * WordPress dependencies
  */
-import { useEntityProp } from '@wordpress/core-data';
-import { useSelect } from '@wordpress/data';
+import { useEntityProp, store as coreStore } from '@wordpress/core-data';
+import { select, dispatch } from '@wordpress/data';
```

Comment:
We should avoid using global select/dispatch and always rely on useSelect/useDispatch. (we might not be targeting the right store with the global select/dispatch)